lib/logstash/inputs/gelf.rb

```diff
@@ -33,6 +33,13 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # ports) may require root to use.
   config :port, :validate => :number, :default => 12201
 
+  # The GELF protocol (TCP or UDP).
+  config :protocol, :validate => :string, :default => "UDP"
+
+  # Whether to capture the hostname or numeric address of the incoming connection
+  # Defaults to hostname
+  config :use_numeric_client_addr, :validate => :boolean, :default => false
```

Comment:
prefer not abbreviating words in config settings, use `address` instead of `addr`